connector-builder-server/src/main/openapi/openapi.yaml

```diff
@@ -0,0 +1,243 @@
+        slices:
+          type: array
+          description: The stream slices returned from the read command
+          items:
+            type: object
+            required:
+              - sliceDescriptor
+              - pages
+            properties:
+              sliceDescriptor:
+                type: object
+                description: 'An object describing the current slice, e.g. {start_time: "2021-01-01", end_time: "2021-01-31"}'
+              pages:
+                type: array
+                description: The pages returned from the read command
+                items:
+                  type: object
+                  required:
+                    - airbyteMessages
+                    - request
+                    - response
+                  properties:
+                    airbyteMessages:
+                      type: array
+                      description: The RECORD/STATE/LOG AirbyteMessages coming from the read operation for this page
+                      items:
+                        $ref: "#/components/schemas/AirbyteProtocol/definitions/AirbyteMessage"
+                    request:
+                      $ref: "#/components/schemas/HttpRequest"
+                    response:
+                      $ref: "#/components/schemas/HttpResponse"
+    StreamReadRequestBody:
+      type: object
+      required:
+        - definition
+        - stream
+      properties:
+        definition:
+          $ref: "#/components/schemas/ConnectorDefinitionBody"
+          description: The config-based connector definition contents
+        stream:
+          type: string
+          description: Name of the stream to read
+        numPages:
+          type: integer
+          description: Number of pages to read from the source for each slice
+          default: 1
+        state:
+          type: object
+          description: State blob to use for incremental streams
+    HttpRequest:
+      type: object
+      required:
+        - url
+      properties:
+        url:
+          type: string
+          description: URL that the request was sent to
+        parameters:
+          type: object
+          description: The request parameters that were set on the HTTP request, if any
+        body:
+          type: object
+          description: The body of the HTTP request, if present
+        headers:
+          type: object
+          description: The headers of the HTTP request, if any
+    HttpResponse:
+      type: object
+      required:
+        - status
+      properties:
+        status:
+          type: integer
+          description: The status of the response
+        body:
+          type: object
+          description: The body of the HTTP response, if present
+        headers:
+          type: object
+          description: The headers of the HTTP response, if any
+    ConnectorDefinitionBody:
+      $ref: ../../../../airbyte-cdk/python/airbyte_cdk/sources/declarative/config_component_schema.json
```

Comment:
we'll need to make sure to wire up the builds to regen if this referenced file changes

Reply:
@sherifnada Do you have an example or know how that would be done? If not I can make a ticket